pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -155,6 +155,16 @@ func (rs *RevisionSpec) Validate(ctx context.Context) *apis.FieldError {
 	return errs
 }
 
+// ValidateContainerConcurrency	function validates the ContainerConcurrency field
+func (rs *RevisionSpec) ValidateContainerConcurrency(ctx context.Context) *apis.FieldError {
```

Comment:
Because ValidateContainerConcurrency is a method on RevisionSpec, you end up having three copies of this code.

I'd make this an independent function so we can reuse it for {v1alpha1,v1beta1}.RevisionSpec and PodAutoscalerSpec:
```go
func ValidateContainerConcurrency(ctx context.Context, cc int64) *apis.FieldError {
	if cc < 0 || cc > int(RevisionContainerConcurrencyMax) {
		return apis.ErrOutOfBoundsValue(
			cc, 0, RevisionContainerConcurrencyMax, apis.CurrentField)
	}
	return nil
}
```

Reply:
@jonjohnsonjr thanks for the suggestion! I've added this to `pkg/apis/serving/metadata_validation.go`